sunpy/map/mapbase.py

```diff
@@ -149,6 +149,7 @@ def __init__(self, data, header, **kwargs):
         # Validate header
         # TODO: This should be a function of the header, not of the map
         self._validate()
+        self.__shift = Pair(0 * u.arcsec, 0 * u.arcsec)
```

Comment:
The general convention we have inherited from astropy NDData is that "read only" properties have a single underscore in the private name. For the sake of (future) consistency I recommend this is changed from `__shift` to `_shift`